### ISSUE TITLE
Issue with callbackURL

### ISSUE THREAD

**Issue by Enricopv** [auth0/lock#861]
Using lock I explicitly state my redirectURL to be http://localhost:1337/users. Yet whenever I successfully login, I'm taken to http://localhost:1337/login. Is there somewhere I'm not looking that's forcing my site to go to /login? It's particularly strange because it's not one of the allowed callback URL's i've specified (and /users is).

![script](https://cloud.githubusercontent.com/assets/7062889/22728582/960834dc-ed93-11e6-9e23-800edd799a80.png)

I have also hardcoded it in my app.js 
![app](https://cloud.githubusercontent.com/assets/7062889/22728617/cfe9812e-ed93-11e6-8bdb-9419b0b6a417.png)

**Comment by luisrudge** [comment]
Hi! What is probably happening is that you're being redirected from /users to /login by your application. Can you open your chrome dev tools, go to the network tab,  enable "preserve log" and check if you're being redirected to /login or /user from Auth0 (maybe take a print screen as well)?

**Comment by Enricopv** [comment]
Hello, thanks Luis. I did as you suggested and auth0 is using my callback of /users and you were right my app is returning /login. I had some code I didn't quite understand and was implementing incorrectly. My route for /user (below), used this ensureLoggedIn thing which I didn't fully understand. 

![ensure](https://cloud.githubusercontent.com/assets/7062889/22745798/9dae9a94-edd6-11e6-81e8-27ac1717e9b8.png)

Apparently I wasn't using session so my app kept redirecting me back to /login. I've included it now and it works. Thank you.

![session](https://cloud.githubusercontent.com/assets/7062889/22746727/adb09bc4-edd9-11e6-82be-de651402a059.png)
